docs/contributing/upgrading-monorepo.md

```diff
@@ -0,0 +1,73 @@
+# Upgrading monorepo
+
+Typical upgrade sequence should be:
+* when you are updating your docker-compose, you need to rebuild your containers then by using command `docker-compose down` followed by `docker-compose up -d --build`
+* *(Windows, MacOS only)* any changes in `docker-sync.yml` file should follow with `docker-sync stop`, `docker-sync clean` and `docker-sync start` to restart synchronization
+* `composer update`
+* `php phing clean`
+* `php phing db-migrations`
+* if you're experiencing some errors, you can always rebuild application and load demo data with `php phing build-demo-dev`
+
+## [From v7.0.0-beta4 to Unreleased](https://github.com/shopsys/shopsys/compare/v7.0.0-beta4...HEAD)
+- [#651 It's possible to add index prefix to elastic search](https://github.com/shopsys/shopsys/pull/651)
+    - either rebuild your Docker images with `docker-compose up -d --build` or add `ELASTIC_SEARCH_INDEX_PREFIX=''` to your `.env` files in the microservice root directories, otherwise all requests to the microservices will throw `EnvNotFoundException`
+- [#679 webserver container starts after php-fpm is started](https://github.com/shopsys/shopsys/pull/679)
+    - add `depends_on: php-fpm` into `webserver` service of your `docker-compose.yml` file so webserver will not fail on error `host not found in upstream php-fpm:9000`
```

Comment:
I've added `depends_on: php-fpm` to my service like this:

```yml
    webserver:
        image: nginx:1.13-alpine
        container_name: shopsys-framework-webserver
        depends_on: php-fpm
        volumes:
            - .:/var/www/html
            - ./docker/nginx/nginx.conf:/etc/nginx/conf.d/default.conf
        ports:
            - "8000:8080"
```

But after running `docker-compose up -d`, I've got this error message:

```
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.webserver.depends_on contains an invalid type, it should be an array
```

The error message helped me to fix it, but I've felt like I've done what was suggested and it failed immediately. If I haven't run `docker-compose up` right after I've changed the file, I could have ended up wasting much more time on this.


```yml
    webserver:
        image: nginx:1.13-alpine
        container_name: shopsys-framework-webserver
        depends_on:
            - php-fpm
        volumes:
            - .:/var/www/html
            - ./docker/nginx/nginx.conf:/etc/nginx/conf.d/default.conf
        ports:
            - "8000:8080"
```

I suggest using the inline array syntax in the code snippet, so it's functional when copy-pasted:

```suggestion
    - add `depends_on: [php-fpm]` into `webserver` service of your `docker-compose.yml` file so webserver will not fail on error `host not found in upstream php-fpm:9000`
```

Reply:
Or it could be made consistent with the upgrade note for project implementations:

```suggestion
    - make sure your `webserver` service depends on `php-fpm` in your `docker-compose.yml` file so webserver will not fail on error `host not found in upstream php-fpm:9000` ([#679](https://github.com/shopsys/shopsys/pull/679))
```

Reply:
thanks, I'll create a PR :slightly_smiling_face: